finance-portal/templates/deployment.yaml

```diff
@@ -73,5 +77,19 @@ spec:
             name: {{ template "finance-portal.fullname" . }}
         - secretRef:
             name: {{ template "finance-portal.fullname" . }}
+      volumes:
+        - name: {{ template "finance-portal.fullname" . }}-secrets-volume
+          secret:
+            secretName: {{ template "finance-portal.fullname" . }}-certs
+            items:
+              - key: first_key
+                path: first_key.key
+                mode: 511
+              - key: second_key
+                path: second_key.key
+                mode: 511
+              - key: third_key
+                path: third_key.key
+                mode: 511
```

Comment:
Wrap in a conditional block `{{- if .Values.JWSKeys }} .. content .. {{- end }}`